.travis.yml

```diff
@@ -22,5 +22,5 @@ matrix:
   include:
     - python: "3.6"
       script:
-        - "flake8 --exclude=manage.py,./apps/main/migrations/,./core/settings.py"
+        - "flake8 --exclude=manage.py,./apps/main/migrations/,./core/settings.py,./apps/scraper/migrations/"
```

Comment:
Can we set a common rule for all migrations?

Reply:
Yes. Sorry for my mistake. I change only setup.cfg and forgot .travis.yml